set.mm

```diff
@@ -266522,6 +266970,36 @@ U C_ ( N ` { X } ) ) -> ( U = ( N ` { X } ) \/ U = { .0. } ) ) $=
     $}
   $}
 
+  ${
+    $d B x y $.  $d I x y $.  $d R x y $.
+    dflidl2.u $e |- U = ( LIdeal ` R ) $.
+    dflidl2.b $e |- B = ( Base ` R ) $.
+    dflidl2.t $e |- .x. = ( .r ` R ) $.
+    ${
+      $d B x y z $.  $d I z $.  $d R z $.  $d .x. z $.
+      $( Lemma for ~ dflidl2 : a sufficient condition for a set to be a left
+         ideal.  (Contributed by AV, 13-Feb-2025.) $)
+      dflidl2lem $p |- ( ( I e. ( SubGrp ` R )
+                      /\ A. x e. B A. y e. I ( x .x. y ) e. I ) -> I e. U ) $=
+        ( vz csubg cfv wcel cv co wral wa adantr eqid wss wne cplusg subgss c0g
+        c0 subg0cl subgcl ad4ant134 ralrimiva ex ralimdvva imp islidl syl3anbrc
+        ne0d ) GDLMNZAOZBOZEPZGNZBGQACQZRGCUAZGUFUBZUTKOZDUCMZPGNZKGQZBGQACQZGF
+        NUQVCVBCGDIUDSUQVDVBUQGDUEMZGDVJVJTUGUPSUQVBVIUQVAVHABCGUQURCNUSGNRZRZV
+        AVHVLVARVGKGUQVAVEGNVGVKVFGDUTVEVFTZUHUIUJUKULUMACVFDEFGBKHIVMJUNUO $.
+    $}
+
+    $d U x y $.
+    $( Alternate (the usual textbook) definition of a (left) ideal of a ring to
+       be a subgroup of the additive group of the ring which is closed under
+       left-multiplication by elements of the full ring.  (Contributed by AV,
+       13-Feb-2025.) $)
+    dflidl2 $p |- ( R e. Ring -> ( I e. U <-> ( I e. ( SubGrp ` R )
+                               /\ A. x e. B A. y e. I ( x .x. y ) e. I ) ) ) $=
+      ( crg wcel csubg cfv cv co wral wa lidlsubg lidlmcl ralrimivva dflidl2lem
+      jca adantl impbida ) DKLZGFLZGDMNLZAOZBOZEPGLZBGQACQZRZUFUGRZUHULDFGHSUNU
+      KABCGCDEFGUIUJHIJTUAUCUMUGUFABCDEFGHIJUBUDUE $.
```

Comment:
It might be nice to refer to this definition in `df-lidl`.
Your PR title says "(Mathbox only)" but this is in main, right?

Reply:
> It might be nice to refer to this definition in `df-lidl`. 

Good idea!

>Your PR title says "(Mathbox only)" but this is in main, right?

No, not Mathbox only (therefore I did not use the label), but the main theorem(s) are in my mathbox.